hydra-node/src/Hydra/Chain/ZeroMQ.hs

```diff
@@ -162,7 +161,7 @@ createMockChainClient EventQueue{putEvent} tracer = do
   -- txs from the chain. For now, we assume it takes 1 sec to connect.
   catchUpTransactions "tcp://127.0.0.1:56790" onTx tracer
   link =<< async (runChainSync "tcp://127.0.0.1:56789" onTx tracer)
-  threadDelay 1
+  threadDelay 0.1
```

Comment:
This has nothing to do with new prelude right? (No need to remove, just checking.)

Reply:
> This has nothing to do with new prelude right? (No need to remove, just checking.)

It has. This used to be the Control.Concurrent threadDelay which takes int as microsecond, whereas the new prelude threadDelay is from io-sim using DiffTime. So this should be much less in practice, but :shrug: